core/tests/protractor/learnerFlow.js

```diff
@@ -403,9 +399,13 @@ describe('Learner dashboard functionality', function() {
     // 'Test Collection'.
```

Comment:
Why is this test only for deskto?

Reply:
Okay, let me once state what the test does. We go to the creator dashboard and then navigate to the collection editor page by clicking on the "Collections" tab. We then chose a collection and then we chose an exploration to add to the existing collection. 
It is only for desktop because of the following reasons:
1. A user can only add an existing exploration to a collection he has created. For desktop, the user creates a collection. Later on, it adds to the same collection it had created earlier. But for mobile, we load a collection and not create it. Therefore, the user cannot add an existing exploration to the loaded collection since the collection is predefined.
2. This involves the collection editor page, which I think has certain components which are not mobile friendly.
3. Creating and later on, editing a collection involves an admin user and not a super admin. We sign-in as super admin for mobile.
4. Also, the collection editor page where we add an existing exploration says that its in beta. So I am not too sure how it will be affected in case of mobile devices.

Therefore, this was kept under desktop.

Reply:
OK, thanks for explaining. Should we keep this part in protractor_desktop then (and add comments explaining why)?

Reply:
(I think in general stuff in protractor/ is stuff that gets tested on both mobile and desktop.)

Reply:
OK, yep that's fine.